src/content/developers/docs/networks/index.md

```diff
@@ -34,7 +34,14 @@ ETH on testnets has no real value; therefore, there are no markets for testnet E
 
 #### Goerli {#goerli}
 
-Goerli is a proof-of-stake testnet. It is expected to be maintained long-term as a stable testnet for application developers. Before its testnet merge, Goerli was a proof-of-authority testnet.
+Goerli is a proof-of-stake testnet. Goerli will be maintained long-term, and is recommended for stakers to test protocol upgrades and developers who want to interact with a large existing state. Before its testnet merge, Goerli was a proof-of-authority testnet.
+
+[Notable properties](https://blog.ethereum.org/2022/06/21/testnet-deprecation):
```

Comment:
Is this a pseudo-heading for the content below? Feels weird to have a link in it.

Reply:
Not sure we want to make the claim it will be maintained long term. Right now we have a serious supply problem and there is a real scenario where we deprecate it in as soon as 12-18 months.